.eslintrc

```diff
@@ -8,7 +8,13 @@
     },
     "rules": {
         "indent": ["error", 4],
-        "standard/no-callback-literal": "off"
+        "standard/no-callback-literal": "off",
+        "no-restricted-globals": [2, "name"]
```

Comment:
This is to prevent `name` being available without using `window.name` (I had a bug with this)